addons/addon-internal-auth-dep-check/README.md

```diff
@@ -0,0 +1,7 @@
+# Addon for checking if environment is ready to upgrade after internal auth deprecation
+
+The purpose of this add-on is check that the environment is ready for deployment after upgrading to the version of Service Workbench that has deprecated internal authentication. Runs only once after upgrade has successfully been deployed.
```

Comment:
_Runs only once after upgrade has successfully been deployed._

Maybe I overlooked something. Which part of the code ensures this?

Reply:
This was from before we discussed the frequency of this step being executed. Thanks for catching it! I'll fix.